src/components/Modal/Modal.tsx

```diff
@@ -1,18 +1,21 @@
 import { Dispatch, SetStateAction, useState } from 'react';
 import styles from './Modal.module.css';
+import { ITodo } from '../../models/ITodo';
+import { formatDate } from '../../utils/formatDate';
+import uuid from 'react-uuid';
 
 interface ModalProps {
-    onSetIsShownModal: Dispatch<SetStateAction<boolean>>,
     createInputText: string,
-    onHandleInputText: any,
-    onAddTodo: any,
-    onFormatDate: any,
+    onHandleInputText: (e: React.FormEvent<HTMLInputElement>) => void,
+    onAddTodo: (todo: ITodo) => void,
+    onSetIsShownModal: Dispatch<SetStateAction<boolean>>,
     onSetCreateInputText: Dispatch<SetStateAction<string>>
 };
 
-export default function Modal({onSetIsShownModal, createInputText, onHandleInputText, onAddTodo, onFormatDate, onSetCreateInputText}: ModalProps) {
-    const [created] = useState<string>(onFormatDate(new Date()));
+export default function Modal({createInputText, onHandleInputText, onAddTodo, onSetIsShownModal, onSetCreateInputText}: ModalProps) {
+    const [created] = useState<string>(formatDate(new Date()));
     const [expires, setExpires] = useState<string>('');
+    const todayMinDate = new Date().toISOString().slice(0, 16);
```

Comment:
Magic number

Reply:
This is a variable to disabling past dates in the input. Today we discussed magic numbers and tou said that they should be variables and not just in the code. Please, tell me how to do it